cusub_cortex/state_machine/config/mission_config_generated.yaml

```diff
@@ -0,0 +1,59 @@
+manager_task_status_pub_rate: 1
```

Comment:
Let's not commit this file, better to add it to .gitignore if it's not already there